pkg/serverless/appsec/httpsec/tags.go

```diff
@@ -78,6 +78,11 @@ type span interface {
 // be in service entry span when AppSec is enabled.
 func setAppSecEnabledTags(span span) {
 	span.SetMetricsTag("_dd.appsec.enabled", 1)
+	value := os.Getenv("DD_APM_TRACING_ENABLED")
+	standalone := value != "true" && value != "1"
+	if standalone {
+		span.SetMetricsTag("_dd.apm.enabled", 0)
```

Comment:
Must be set in the trace chunk tags now.

Reply:
Done in e8da65f67931ed9b